demo/src/androidTest/java/com/novoda/movies/rateable/RateableMovieViewModelFixtures.java

```diff
@@ -0,0 +1,16 @@
+package com.novoda.movies.rateable;
+
+import com.novoda.movies.R;
+
+class RateableMovieViewModelFixtures {
+
+    static RateableMovieViewModelBuilder viewModel(RateableMovieViewModel.UserActions userActions) {
+        return viewModel().actions(userActions);
+    }
+
+    static RateableMovieViewModelBuilder viewModel() {
```

Comment:
the fixtures don't return the viewmodel directly, but the builder instead, giving us the opportunity to modify the values in the tests, without having to specify bits we don't care about